usaspending_api/disaster/v2/views/award/amount.py

```diff
@@ -41,16 +42,28 @@ def post(self, request):
 
     @property
     def queryset(self):
+        return self._file_d_queryset() if self.award_type_codes else self._file_c_queryset()
+
+    def _file_d_queryset(self):
```

Comment:
Minor gripe on the three method names: if the response is a queryset with a .aggregate() then it isn't returning a queryset

Reply:
Fair point. Will update them